core/vm/access_list_tracer.go

```diff
@@ -137,7 +137,7 @@ func NewAccessListTracer(acl types.AccessList, from, to common.Address, precompi
 	}
 }
 
-func (a *AccessListTracer) CaptureStart(env *EVM, from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int) {
+func (a *AccessListTracer) CaptureStart(env *EVM, from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int, authorizationResults []types.AuthorizationResult) {
 }
```

Comment:
Technically we should capture the set code state accesses here. But it's probably fine if we don't.

Reply:
added in https://github.com/scroll-tech/go-ethereum/pull/1112/commits/eeafee422b8f158d214788b5a6c6f127f8fda60f.